mobile/lib/main.dart

```diff
@@ -424,3 +369,64 @@ Future<void> logAppSettings(bridge.Config config) async {
   String nodeId = rust.api.getNodeId();
   FLog.info(text: "Node ID: $nodeId");
 }
+
+/// Forward the events from change notifiers to the Event service
+void subscribeToNotifiers(BuildContext context) {
```

Comment:
I like that we pass the `BuildContext` here, that enforces that we can only call this one the build context is available. Makes it more clear when this is needed.